org.flatpak.rmac.yml

```diff
@@ -0,0 +1,23 @@
+id: org.flatpak.rmac
```

Comment:
this is definitely wrong. This is not a Flatpak project.

Reply:
this wasn't addressed. and the reason why it is blocked.

The checklist isn't a CAPTCHA:

> The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines](https://docs.flathub.org/docs/for-app-authors/requirements#application-id) are followed.

Reply:
My apologies, I didn't edit the ID but only the file. I'm still learning about flatpak submission, so I thank you for your patience.